include/aikido/common/detail/parseSkeleton-impl.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef AIKIDO_COMMON_DETAIL_PARSESKELETON_IMPL_HPP_
```

Comment:
Please move this to src/common/parseSkeleton.cpp.

Header impl's are supposed to be for template classes (which cannot be pre-compiled into the library). Functions like this don't have that issue, and will unnecessarily slow down the compilation time of all sub-libraries that use it.

Reply:
Thanks! Yeah I wasn't sure what requires putting some code in an impl and others not -- this helps.